platform/web-security/src/main/java/org/motechproject/security/email/impl/EmailSenderImpl.java

```diff
@@ -112,6 +139,25 @@ public void sendLoginInfo(final MotechUser user, String token) {
         return params;
     }
 
+    private Map<String, Object> templateParams(MotechUser user, DateTime expirationDate) {
```

Comment:
Maybe instead of just overloading this method multiple times, extend the name a bit so it is more clear for what it's used. Example: passExpirationTemplateParams()